tests/e2e/volumecreate_test.go

```diff
@@ -67,6 +72,7 @@ var _ = Suite(&VolumeCreateTestSuite{})
 // 6. contains multiple '@'
 // 7. contains unicode character
 func (s *VolumeCreateTestSuite) TestValidName(c *C) {
+	log.Printf("START: VolumeCreateTestSuite.TestValidName")
```

Comment:
Nit: in existing tests we are using the file name (e.g. "START: restart_test.TestVolumeStaleMount"). I'm fine with both styles, just keep them consistent.